client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.scss

```diff
@@ -19,3 +19,19 @@
 .google-workspace-icon {
 	max-width: 36px;
 }
+
+.google-workspace-card__price-badge {
+	.promo-card__price {
+		.price__cost {
+			.google-workspace-card__interval, .google-workspace-card__mailbox {
+				font-size: 0.875rem;
+				color: var( --studio-gray-80 );
+			}
+			.google-workspace-card__sale {
+				color: var( --studio-green-50 );
+			}
+		}
+	}
+}
```

Comment:
These CSS selectors were removed by my changes to the `PriceWithInterval` component yesterday. I think we should remove this block entirely.